erpnext_documentation/www/docs/user/manual/en/setting-up/users-and-permissions/role-based-permissions.md

```diff
@@ -51,7 +51,10 @@ Leave Application is a good example that encompasses all areas of a Permission S
   Hence, User Permissions record should be created for each User-Employee combination.
 
   <img class="screenshot" alt="Limiting access to Leave Applications for a user with Employee Role via User Permissions Manager" src="/docs/assets/img/users-and-permissions/setting-up-permissions-employee-user-permissions.png">
-  
+
+* If you want that an **Employee** should only have select right on **Leave Type** then grant only `Select` perm to role Employee
```

Comment:
```suggestion
*If you want an **Employee** to only select a document in another document and not have read access to that document as a whole, then grant only `Select` perm to the role, Employee.
```

Reply:
explain this example (regarding leave type, how the employee will not be able to access the leave type doctype via leave application but will be able to select it in the leave application using select perm)